draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -437,9 +442,9 @@ External operations:
 ~~~
 ExternalMu-ML-DSA.Prehash(pk, M, ctx):
 
-  if |ctx| > 255 then
+  if |ctx| > 0 then
     return error  # return an error indication if the context string is
-                  # too long
+                  # not the empty string
```

Comment:
These two lines should not be modified as these comments are still useful in case we want to do "prehashing" in a use case where context string is allowed.

Reply:
Done